engine/pom.xml

```diff
@@ -388,6 +388,18 @@
       <version>1.6.5</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.sun.jersey.jersey-test-framework</groupId>
+      <artifactId>jersey-test-framework-grizzly2</artifactId>
+      <version>1.9</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>servlet-api</artifactId>
```

Comment:
this artifact is pulled in by hadoop-common:test and many other dependencies, but it conflicts with the artifact org.eclipse.jetty:jetty-servlet:jar during unit tests. That's why I needed to mark it provided so it won't be pulled in during the running of tests